Tests/DatadogTests/Datadog/Mocks/SystemFrameworks/FoundationMocks.swift

```diff
@@ -230,6 +230,10 @@ extension String: AnyMockable, RandomMockable {
         )
     }
 
+    static func mockAnySource() -> String {
+        return ["ios", "android", "browser", "ios", "react-native", "flutter"].randomElement()!
+    }
```

Comment:
This is rather RUM-specific mock than Foundation mock, hence `FoundationMocks` is IMO not the best place for it. How moving this next to existing `RUMErrorEvent.Error.SourceType` mock in `RUMDataModelMocks.swift`?

https://github.com/DataDog/dd-sdk-ios/blob/18031501d1b6e8731360b1db8f644b193e039491/Tests/DatadogTests/Datadog/Mocks/RUMDataModelMocks.swift#L225

Following this segregation will pay off nicely when we'll be extracting modules (both in implementation and tests).

Reply:
Yeah, good call. I'll move it.